src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -200,18 +200,18 @@ static void regclass_graph_PostProcessSteps(py::module m) {
 
     steps.def(
         "convert_element_type",
-        [](ov::preprocess::PostProcessSteps& self, ov::element::Type type) {
+        [](ov::preprocess::PostProcessSteps& self, ov::element::Type type = {}) {
             return &self.convert_element_type(type);
         },
-        py::arg("type"),
+        py::arg("type") = ov::element::Type{},
```

Comment:
Same here `help(openvino.preprocess.PreProcessSteps.convert_element_type)`, just a different overload.

Reply:
Help on instancemethod in module openvino.pyopenvino.preprocess:

convert_element_type(...)
    convert_element_type(self: openvino.pyopenvino.preprocess.PostProcessSteps, type: openvino.pyopenvino.Type = <Type: 'uundefined18446744073709551615'>) -> openvino.pyopenvino.preprocess.PostProcessSteps
    
    
    Converts tensor element type to specified type.
    Tensor must have openvino.Type data type.
    
    Parameters
    ----------
    type : Type
        Destination type. By default type will be taken from model output's element type.
    
    Returns
    ----------
    convert_element_type : PostProcessSteps
        Reference to itself to allow chaining of calls in client's code in a builder-like manner.

Reply:
Corrected output looks like:
```
Help on instancemethod in module openvino.pyopenvino.preprocess:

convert_element_type(...)
    convert_element_type(self: openvino.pyopenvino.preprocess.PostProcessSteps, type: openvino.pyopenvino.Type = {}) -> openvino.pyopenvino.preprocess.PostProcessSteps
    
    
    Converts tensor element type to specified type.
    Tensor must have openvino.Type data type.
    
    Parameters
    ----------
    type : Type
        Destination type. If not specified, type will be taken from model output's element type.
    
    Returns
    ----------
    convert_element_type : PostProcessSteps
        Reference to itself to allow chaining of calls in client's code in a builder-like manner.
```

Reply:
Fixed